scripts/setupLocalIdler.sh

```diff
@@ -150,6 +152,28 @@ forwardToggleService() {
     echo "Toggle port forward stopped." >&2
 }
 
+###############################################################################
+# Retrieves the required OpenShift and Auth token from
+# Globals:
+#   JC_OPENSHIFT_API_TOKEN - OpenShift service account token used to monitor
+#                            OpenShift events
+#   JC_AUTH_TOKEN          - Auth token for the auth service
+# Arguments:
+#   None
+# Returns:
+#   None
+###############################################################################
+setTokens() {
+    pod=$(loc get pods -l deploymentconfig=jenkins-idler -o json | jq -r '.items[0].metadata.name')
```

Comment:
@hferentschik `mv loc oc` ?

Reply:
If you look at the whole script, `loc` is a function wrapper for `oc`